explainer.md

```diff
@@ -275,29 +324,59 @@ Acquire performs the following steps:
 
 Note that the acquire call takes options, consisting of the following key-value
 pairs:
-* (optional) timeout: timeoutInMilliseconds
+* (optional) `timeout`: `timeoutInMilliseconds`
   * This is an optional timeout value for the duration of the lock. When the
-    lock is acquired, and commit is not called within timeoutInMilliseconds
+    lock is acquired, and commit is not called within `timeoutInMilliseconds`
     milliseconds, the lock will be automatically committed. This is a safety
     valve for protecting pages from inadvertently leaving an element locked
     indefinitely due to uncaught exceptions or other bugs in the code. For a
     more sophisticated use cases, the value of Infinity will effectively disable
     the timeout.
+* (optional) `activatable`: `true|false`
+  * Signifies whether the user-agent can automatically commit the lock
+    when the browser needs to *activate* the locked element or its descendants.
+    See [element activation section](#element-activation) for more details.
+  * Not setting this, or setting this to `false`,
+    means this element and its descendants will be ignored in cases like
+    find-in-page, anchor link navigation, focus navigation, etc.
+
+#### DisplayLockContext.locked
+
+This is a boolean value that will be `true`
+if the `Element` associated with this `DisplayLockContext` is in the locked state,
+and `false` if not.
+
+#### The locked state
+
+#### What happens
+
+When its lock is acquired, if the element already existed in the DOM,
+the visual content that was present at the time the lock was acquired is cleared,
```

Comment:
This text about visibility: hidden is ok for now, but just noting that the discussion
we had on Monday about intrinsic sizing has bearing on it.

However, there is a problem in your description: the root element being locked in this
case is still painted and laid out. It's the subtree that is not painted.

Reply:
Hmm yeah, actually removed the `visibility:hidden` part now, we should update the details about intrinsic sizing later.